CONTRIBUTING.md

```diff
@@ -61,35 +61,64 @@ In addition to those general ideas, we use the following conventions:
     this has severe performance implications due to calling constructors.
 
 ### Documentation + Tests
-  * All definitions of classes etc. should be documented using doxygen
-  * All new features should have an usage example in the wiki
+  * All definitions of classes, etc, should be documented using Doxygen.
+  * All new features should have an usage example in the wiki.
```

Comment:
The examples are now included in the documentation and no longer in the wiki. We should update this:
"All new features should have an usage example included in the documentation"